output.R

```diff
@@ -83,17 +83,21 @@ if ("--help" %in% flags) {
   q()
 }
 
-choose_slurmConfig_output <- function(slurmExceptions = NULL) {
+choose_slurmConfig_output <- function(output) {
   slurm_options <- c("--qos=priority", "--qos=short", "--qos=standby",
                      "--qos=priority --mem=8000", "--qos=short --mem=8000",
                      "--qos=standby --mem=8000", "--qos=priority --mem=32000")
+  
+  # Modify slurm options for ar6 reporting, since we want to run MAGICC in parallel and we'll need a lot of memory
+  if ("ar6Climate" %in% output) slurm_options <- paste(slurm_options[1:3], "--tasks-per-node=12 --mem=32000")
 
   if (!isSlurmAvailable())
     return("direct")
 
   if (!is.null(slurmExceptions)) {
```

Comment:
something went missing here. `slurmExceptions` is never defined, and the `    slurmExceptions <- if ("reporting" %in% output) "--mem=[0-9]*[0-9]{3}" else NULL` part deleted below is not reproduced anymore.

Reply:
Yea I'm afraid messed up the merge here. Imma fix it